.kitchen.yml

```diff
@@ -31,6 +31,15 @@ platforms:
   - name: windows-2012r2
     driver_config:
       box: chef/windows-server-2012r2-standard
+# Solaris 10 requires the OS ISO to be mounted into `/cdrom` in order for the
```

Comment:
I don't have any objections to doing this, just note that doing so will add ~1.7GB to the Solaris 10  vagrant box. Since we wouldn't be "redistributing" this and it is a private config, we probably can just copy the `/Solaris_10/Product` directory into `/cdrom` on the Atlas box rather than the ISO itself.